Makefile

```diff
@@ -0,0 +1,23 @@
+# url to a running release-manager
+URL=localhost:8080
+FILE=payload.json
+PAYLOAD=`cat $(FILE)`
+
+github-webhook:
+	curl -H 'X-GitHub-Event: pull_request' \
+	-H 'Content-Type: application/json' \
+	-d '$(shell cat ${FILE})' \
+	$(URL)/webhook/github
+
+
+
+github-webhook-jacob:
```

Comment:
Is this make target still relevant or did we set for `github-webhook`?

Reply:
No it's not relevant anymore. I removed it